packages/turndown/src/commonmark-rules.js

```diff
@@ -266,6 +266,9 @@ rules.fencedCodeBlock = {
 
     var fence = repeat(fenceChar, fenceSize)
 
+    // delete code number lines
+    code = code.replace(/(^-\s+\d+\s*$\r?\n)+/gm, '');
+
```

Comment:
I'm concerned that this may also remove information that shouldn't be removed from certain code blocks. In the comment above could clarify exactly what strings are being removed? (as I'm unable to read that regex)

Reply:
This regular expression removes consecutive lines of code that start with a - followed by a number. The regular expression marker `/gm ` means that each line break is separated by a line, and can be perfectly matched with a line using` (^ - \ s+\ d+\ s * $\ r? \ n) `

Reply:
Ref: [RegExp.prototype.multiline](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/multiline)

Reply:
@laurent22  Match the lines: 
![image](https://github.com/laurent22/joplin/assets/88243938/6f0e990f-1cb4-408e-82e8-f4cdd8d38c4e)

Reply:
Continuous lines of code starting with - followed by numbers are basically useless information.

Reply:
It might make more sense to ignore code-block children of type `ul` with class `pre-numbering`. This should also match line numbers in the second example provided.

This would allow the web clipper (and perhaps also the Rich Text Editor) to save blocks that contain lists of numbers, while also removing line numbers.

Reply:
Thank you for your patient guidance. I will try to modify the code using the method you mentioned.

Reply:
@personalizedrefrigerator   Already edit rules.list as your guidance. Test units run ok. Thanks a lot.

Reply:
> I'm concerned that this may also remove information that shouldn't be removed from certain code blocks. 

@laurent22   This regular replacement code has been removed.